sql/hive/src/test/resources/golden/udf_bitwise_xor-1-8fc9b9bf1aced42754419310784e0a9f

```diff
@@ -1,4 +1,4 @@
 a ^ b - Bitwise exclusive or
 Example:
   > SELECT 3 ^ 5 FROM src LIMIT 1;
-  2
+  6
```

Comment:
Surprisingly, this test case was added at 3a845d3c048eebb0bddb3937128746fde3e8e4d8 .

Reply:
ancient test case ..